pom.xml

```diff
@@ -516,7 +516,7 @@
                         <dependency>
                             <groupId>org.apache.maven.shared</groupId>
                             <artifactId>maven-dependency-analyzer</artifactId>
-                            <version>1.11.1</version>
+                            <version>1.11.2</version>
```

Comment:
I think we can drop this, because this issue was fixed in `maven-dependency-plugin` version 3.1.2, which we're now using.